integration/onewatch_test.go

```diff
@@ -51,4 +43,19 @@ func TestOneWatch(t *testing.T) {
 	// Assert that the pods were changed in-place, and not that we
 	// created new pods.
 	assert.Equal(t, oneUpPods, twoUpPods)
+
+	// Delete the pod and make sure it got replaced with something that prints the
+	// same thing (crash rebuild).
```

Comment:
nbd but I think this comment should go above the `runCommandSilently` call instead of here (made me think the length check was somehow deleting pods for a sec)

Reply:
done!